From 7e9f9c5ea4b001d4f08330f87dc67b8bb8e14eb1 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 12 Jan 2016 14:45:03 +0100
Subject: [PATCH] Added 'zero row' to the end of the long opts array

---
 tcpdump.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tcpdump.c b/tcpdump.c
index a63bacc..9a0761c 100644
--- a/tcpdump.c
+++ b/tcpdump.c
@@ -445,6 +445,7 @@ show_dlts_and_exit(pcap_t *pd)
 
 static struct option longopts[] = {
 	{ "time-stamp-precision", required_argument, NULL, OPTION_TSTAMP_PRECISION},
+  { NULL, 0, 0, 0 },
 };
 
 #ifndef WIN32
-- 
2.5.0

